pkg/manifests/manifests_test.go

```diff
@@ -851,16 +851,46 @@ func TestPrometheusOperatorConfiguration(t *testing.T) {
 func getContainerArgValue(containers []v1.Container, argFlag string, containerName string) string {
 	for _, container := range containers {
 		if container.Name == containerName {
-			for _, arg := range container.Args {
-				if strings.HasPrefix(arg, argFlag) {
-					return arg
-				}
+			arg, ok := getArgValue(container, argFlag)
+			if ok {
+				return arg.original
 			}
 		}
 	}
 	return ""
 }
 
+type argValue struct {
+	key      string
+	value    string
+	original string
+}
+
+// returns the value of the container arg if found or false
+func getArgValue(container v1.Container, flag string) (argValue, bool) {
+	for _, arg := range container.Args {
+		if strings.HasPrefix(arg, flag) {
+			parts := strings.Split(arg, "=")
+			return argValue{
+				key:      parts[0],
+				value:    parts[1],
+				original: arg,
+			}, true
+		}
+	}
+	return argValue{}, false
+}
+
+// returns the v1.EnvVar of the container if a match is found  or false
+func getEnvValue(container v1.Container, name string) (*v1.EnvVar, bool) {
```

Comment:
it doesn't seem to be used, probably a left-over from a previous version?

Reply:
correct